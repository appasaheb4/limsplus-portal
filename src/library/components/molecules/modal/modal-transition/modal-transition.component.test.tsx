import React from 'react';
import {render} from '@utils';
import {ModalTransition} from './moda-transition.component';

it('render modalTransition correctly', () => {
  const modalTransition = render(
    <ModalTransition show={true} onClick={() => jest.fn()} />,
  );
  expect(modalTransition).toMatchSnapshot();
});
