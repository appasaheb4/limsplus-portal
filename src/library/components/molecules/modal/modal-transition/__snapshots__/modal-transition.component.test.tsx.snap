// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render modalTransition correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <div
          class="fixed inset-0 overflow-hidden z-50 false"
        >
          <div
            class="absolute inset-0 overflow-hidden"
          >
            <div
              class="ease-in-out duration-500 opacity-0"
            >
              <div
                aria-hidden="true"
                class="absolute inset-0 bg-gray-600 bg-opacity-75 transition-opacity"
              />
            </div>
            <section
              aria-labelledby="slide-over-heading"
              class="absolute inset-y-0 right-0 pl-10 max-w-full flex"
            >
              <div
                class="transform transition ease-in-out duration-500 sm:duration-700 translate-x-full"
              >
                <div
                  class="relative w-screen max-w-xs h-full"
                >
                  <div
                    class="absolute top-0 left-0 -ml-8 pt-4 pr-2 flex sm:-ml-10 sm:pr-4"
                  >
                    <button
                      class="rounded-md text-gray-300 hover:text-white focus:outline-none focus:ring-2 focus:ring-white"
                    >
                      <span
                        class="sr-only"
                      >
                        Close panel
                      </span>
                      <svg
                        aria-hidden="true"
                        class="h-6 w-6"
                        fill="none"
                        stroke="currentColor"
                        viewBox="0 0 24 24"
                        xmlns="http://www.w3.org/2000/svg"
                      >
                        <path
                          d="M6 18L18 6M6 6l12 12"
                          stroke-linecap="round"
                          stroke-linejoin="round"
                          stroke-width="2"
                        />
                      </svg>
                    </button>
                  </div>
                  <div
                    class="h-full flex flex-col bg-white shadow-xl overflow-y-scroll"
                  />
                </div>
              </div>
            </section>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <div
        class="fixed inset-0 overflow-hidden z-50 false"
      >
        <div
          class="absolute inset-0 overflow-hidden"
        >
          <div
            class="ease-in-out duration-500 opacity-0"
          >
            <div
              aria-hidden="true"
              class="absolute inset-0 bg-gray-600 bg-opacity-75 transition-opacity"
            />
          </div>
          <section
            aria-labelledby="slide-over-heading"
            class="absolute inset-y-0 right-0 pl-10 max-w-full flex"
          >
            <div
              class="transform transition ease-in-out duration-500 sm:duration-700 translate-x-full"
            >
              <div
                class="relative w-screen max-w-xs h-full"
              >
                <div
                  class="absolute top-0 left-0 -ml-8 pt-4 pr-2 flex sm:-ml-10 sm:pr-4"
                >
                  <button
                    class="rounded-md text-gray-300 hover:text-white focus:outline-none focus:ring-2 focus:ring-white"
                  >
                    <span
                      class="sr-only"
                    >
                      Close panel
                    </span>
                    <svg
                      aria-hidden="true"
                      class="h-6 w-6"
                      fill="none"
                      stroke="currentColor"
                      viewBox="0 0 24 24"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M6 18L18 6M6 6l12 12"
                        stroke-linecap="round"
                        stroke-linejoin="round"
                        stroke-width="2"
                      />
                    </svg>
                  </button>
                </div>
                <div
                  class="h-full flex flex-col bg-white shadow-xl overflow-y-scroll"
                />
              </div>
            </div>
          </section>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
